jenkinsfiles/UB20_Check.groovy

```diff
@@ -80,7 +80,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
         def DATESTEMP = sh(script: "date +\"%Y-%m-%d\"", returnStdout: true).trim()
```

Comment:
DATESTEMP should be changed to DATESTAMP to be consistent if the timestamp change is made.